tests/test_conformity_type.py

```diff
@@ -0,0 +1,49 @@
+from __future__ import absolute_import, unicode_literals
+
+import unittest
+
+from conformity.fields import (
+    IPAddress,
+    Longitude
+)
+
+
+class ConformityTypeTests(unittest.TestCase):
+    """
+    Tests introspection output of conformity_type
+    """
+
+    def test_returns_conformity_type_if_asked_upon_introspection(self):
+        schema = Longitude()
+
+        detailed_introspect = schema.introspect(include_conformity_type=True)
+
+        self.assertTrue("conformity_type" not in schema.introspect())
+        self.assertTrue("conformity_type" in detailed_introspect.keys())
+        self.assertEqual(
+            detailed_introspect["conformity_type"],
+            Longitude.conformity_type
+        )
+        self.assertEqual(
+            detailed_introspect["type"],
+            Longitude.introspect_type
+        )
+        # Note: Specific to some types, others actually do conformity_type = type
+        self.assertNotEqual(
+            detailed_introspect["type"],
+            detailed_introspect["conformity_type"]
+        )
+
+    def test_ipaddress_introspect(self):
+        schema = IPAddress()
```

Comment:
Because partial objects are a in a python limbo where both `inspect.isclass()` and `inspect.isfunction()` return `False`, for this cases to perform the introspection and correctly have `conformity_type` an instance is needed.

This has no effect on Conformity usage itself, and on my contract testing tool I have a way of programmatically detecting partial objects and creating instances of them to call `introspect(include_conformity_type=True)` etc.